src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -105,6 +139,96 @@ boot_newcomp_sinv_alloc(spdid_t spdid)
 	}
 }
 
+/* TODO: possible to cos_defcompinfo_child_alloc if we somehow move allocations to one place */
+static void
+boot_newcomp_defcinfo_init(spdid_t spdid)
```

Comment:
All of these names are kind of blurring together.  I think we lost structure in naming long ago, but it is making understanding all of this very difficult.